src/sdk/model/vdc/distributed-firewall/distributed-firewall-layer2-update-request.ts

```diff
@@ -0,0 +1,97 @@
+
+/* istanbul ignore next: autogenerated */
```

Comment:
This autogenerated comment should be above the exported class

Reply:
Good catch, thank you.